shared/chat/conversation/bottom-banner/container.tsx

```diff
@@ -41,11 +39,7 @@ const mapStateToProps = (state, {conversationIDKey}) => {
   }
 }
 
-const mapDispatchToProps = dispatch => ({
-  onClick: isMobile
-    ? (username: string) => dispatch(ProfileGen.createShowUserProfile({username}))
-    : (username: string) => dispatch(Tracker2Gen.createShowUser({asTracker: true, username})),
-})
+const mapDispatchToProps = dispatch => ({})
 
 const mergeProps = (stateProps, dispatchProps) => {
```

Comment:
Nit: you can delete the `dispatchProps` parameter. Also, while you're here, can you move the `mergeProps` to make it typed?

Reply:
Good call. I think it was already moved upstream, so I just rebased.